patches/minecraft/net/minecraft/world/WorldProviderEnd.java.patch

```diff
@@ -0,0 +1,17 @@
+--- ../src-base/minecraft/net/minecraft/world/WorldProviderEnd.java
++++ ../src-work/minecraft/net/minecraft/world/WorldProviderEnd.java
+@@ -16,6 +16,14 @@
+ {
+     private DragonFightManager field_186064_g = null;
+ 
++    /*======================================= Forge Start =========================================*/
++    public WorldProviderEnd()
```

Comment:
This is necessary - this can't be in the getter method because this message could be changed using the setter, and having another variable to check if it's been changed or not is just plain ugly.